### PR TITLE
fix(css): update 'display' interactive demo

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- addresses https://github.com/mdn/interactive-examples/pull/2710#issuecomment-1888114868

### CODE REVIEW

File: files/en-us/web/css/display/index.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The **`display`** [CSS](/en-US/docs/Web/CSS) property sets whether an element is
 
 Formally, the **`display`** property sets an element's inner and outer _display types_. The outer type sets an element's participation in [flow layout](/en-US/docs/Web/CSS/CSS_flow_layout); the inner type sets the layout of children. Some values of `display` are fully defined in their own individual specifications; for example the detail of what happens when `display: flex` is declared is defined in the CSS Flexible Box Model specification.
 
+In the following demo the orange element represents the `div` on which different `display` values are set. It contains three child elements.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:14]:
```suggestion
In the following demo the orange element represents the `<div>` on which different `display` values are set. It contains three child elements.
```

File: files/en-us/web/css/display/index.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ The **`display`** [CSS](/en-US/docs/Web/CSS) property sets whether an element is
 
 Formally, the **`display`** property sets an element's inner and outer _display types_. The outer type sets an element's participation in [flow layout](/en-US/docs/Web/CSS/CSS_flow_layout); the inner type sets the layout of children. Some values of `display` are fully defined in their own individual specifications; for example the detail of what happens when `display: flex` is declared is defined in the CSS Flexible Box Model specification.
 
+In the following demo the orange element represents the `<div>` on which different `display` values are set. It contains three child elements.
```
Comment (estelle) [inline files/en-us/web/css/display/index.md:14]:
```suggestion
In the following demo, the element with the dashed orange border represents the `<div>` on which different `display` values are set. It contains three child elements.
```
I made a suggested change on the example code in the other PR

Comment (estelle) [reply files/en-us/web/css/display/index.md]: (prefer not to use color alone b/c protanopia)

**Review by estelle — APPROVED** [review]

**Comment by estelle** [comment]
approved, but not merging until 2710 is merged.

**Comment by OnkarRuikar** [comment]
The description was added in demo itself.

**Pull request closed without merging**